### ISSUE TITLE
CSS selector works in verifyElementPresent but not in waitForElementPresent

### ISSUE THREAD

**Issue by cawoodm** [SeleniumHQ/selenium-ide#455]
The selector `css=ul.part-list` works in `verifyElementPresent` but not in `waitForElementPresent`:

![image](https://user-images.githubusercontent.com/498834/47134308-7beae680-d2ac-11e8-8790-775e336b4009.png)

As you can see it does exist:
![image](https://user-images.githubusercontent.com/498834/47134293-7097bb00-d2ac-11e8-881e-645c2bdb1b5a.png)

And run via the webdriver (.NET) it gets selected fine.

Windows 10 / Chrome 67

**Comment by 98gmarquee** [comment]
> 
> 
> `wait for element present` takes two arguments 1) a locator (target) and 2) a timeout in milliseconds (value). The log is telling you this -- `Timeout not specified`. You can see the details of the command in the `Reference` tab.

Yep, that caught me up too recently after importing. Might be something to look at to put a default of 300000 or something in the value when importing scripts for old IDE. Took me a while to go through and add a value in there. I usually set mine for 5 minutes, 300000.

**Comment by tourdedave** [comment]
Having a sensible default is an interesting idea. But 5 minutes? That seems like *a lot*. Why so long?

**Comment by 98gmarquee** [comment]
Just to avoid issues when having things calling 3rd party sites that take time to load. Usually 30 seconds is all it takes, but we do have things in our system that takes minutes the first time for the data to cache on the web servers. Then after that it's nearly instant. So, with a 5 minute wait, my scripts don't stall out the first time.

**Comment by dcanning** [comment]
How does one specify the wait timeout using the selenium IDE?  I tried editing the project file json to include "waitTime":"30000" on the test step, but that did nothing.

**Comment by tourdedave** [comment]
@dcanning Open questions like this are better suited for the selenium-users group or the Selenium chat channel. Details for them are available on [the project's help page](https://www.seleniumhq.org/selenium-ide/help/).

For reference, the explicit wait commands are documented [here](https://www.seleniumhq.org/selenium-ide/docs/en/api/commands/#wait-for-element-editable).
